ext/hiera/hiera.yaml

```diff
@@ -0,0 +1,14 @@
+---
+version: 5
+defaults:
+  # datadir is empty here, so hiera uses its defaults:
+  # - /etc/puppetlabs/code/environments/%{environment}/hieradata on *nix
+  # - %CommonAppData%\PuppetLabs\code\environments\%{environment}\hieradata on Windows
+  # When specifying a datadir, make sure the directory exists.  datadir:
```

Comment:
The information about `datadir` is not correct. By default it is a directory named ´data´ in the root directory of the environment (i.e. next to the `hiera.yaml` file).
The text that says `/etc/puppetlabs/code/environments/%{environment}/hieradata` is not correct since `/etc/puppetlabs/code/environments` depends on a setting. And ´hieradata´ is the version 3 standard. The Windows equiv has the same problem.